### PR TITLE
feat(1610): Add parseInt to terminationgraceperiod

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
env var

## Context
The env var from k8s is type string for terminationGracePeriodSceonds

## Objective

This adds a parseInt to the env var

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: lib/registerPlugins.js [hunk 1]
```diff
@@ -41,7 +41,10 @@ function registerCustomPlugin(server, callback) {
     async.eachSeries(['shutdown'], (pluginName, next) => {
         server.register({
             register: require(`../plugins/${pluginName}`),
-            options: { terminationGracePeriod: process.env.TERMINATION_GRACE_PERIOD || 30 }
+            options: {
+                terminationGracePeriod: parseInt(process.env.TERMINATION_GRACE_PERIOD, 2)
```
Comment (jithine) [inline lib/registerPlugins.js]: radix should be 10. Alternatively you can just do `+process.env.TERMINATION_GRACE_PERIOD`

Comment (pritamstyz4ever) [reply lib/registerPlugins.js]: fixed

Comment (parthasl) [reply lib/registerPlugins.js]: why not just Number?

Comment (jithine) [reply lib/registerPlugins.js]: both provide same outcome. https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number

`+` is same as Number()

**Comment by tk3fftk** [comment]
https://github.com/screwdriver-cd/screwdriver/issues/1610

**Pull request merged**